drivers/include/ft5x06.h

```diff
@@ -71,11 +88,12 @@ typedef void (*ft5x06_event_cb_t)(void *arg);
  * @brief   Device initialization parameters
  */
 typedef struct {
-    i2c_t i2c;              /**< I2C device which is used */
-    uint8_t addr;           /**< Device I2C address */
-    gpio_t int_pin;         /**< Touch screen interrupt pin */
-    uint16_t xmax;          /**< Touch screen max X position */
-    uint16_t ymax;          /**< Touch screen max Y position */
```

Comment:
I don't think the doxygen comments should be shifted. This would only result in a one line being added.

Reply:
Reverted in 06bd29c67dc805e2ba924bc208c5ed04284e08f6